### PR TITLE
More primitives

### PR DESCRIPTION
**Pull request by PalumboN**
- Compilation of more primitives.
- Changes in register allocation + new stager

VM: https://github.com/pharo-project/pharo-vm/pull/730

### CODE REVIEW

**Comment by PalumboN** [comment]
Lots of failing tests 👎

File: Druid/DRAbstractCompilerCompiler.class.st [hunk 1]
```diff
@@ -89,9 +89,14 @@ DRAbstractCompilerCompiler >> customiseFor: method [
...
 	method ast
 		pragmaNamed: #customisedReceiverFor:
-		ifPresent: [ :pragma | 
+		ifPresent: [ :pragma |
 			customisation := DRPrimitiveCustomisation newCustomisation:
-				                  pragma arguments first value ]
+				                 pragma arguments first value ].
+
+	method ast
+		pragmaNamed: #checkArgumentCount
```
Comment (guillep) [inline Druid/DRAbstractCompilerCompiler.class.st:97]: interesting

**Review by guillep — APPROVED** [review]
Changes look ok to me, let's look at the tests!

**Comment by PalumboN** [comment]
Most failing tests are for cases that are not related to the current meta-compilation of the VM.

Useless tests, or useless dev strategy.

**Pull request merged**